ui/src/api/src/api/FacetsApi.js

```diff
@@ -45,7 +45,7 @@ export default class FacetsApi {
     /**
      * Returns facets.
      * @param {Object} opts Optional parameters
-     * @param {Array.<String>} opts.filter 
+     * @param {Array.<String>} opts.filter filter represents selected facet values. Elasticsearch query will be run only over selected facet values. filter is an array of strings, where each string has the format \&quot;facetName&#x3D;facetValues\&quot;. Example url /facets?filter&#x3D;Gender&#x3D;female&amp;Region&#x3D;northwest,southwest 
      * @param {module:api/FacetsApi~facetsGetCallback} callback The callback function, accepting three arguments: error, data, response
      * data is of type: {@link module:model/FacetsResponse}
      */
```

Comment:
Can you document somewhere in PR why you regenerated Swagger? It's a bit strange that swagger was regenerated when api.yaml didn't change.

Reply:
I had updated the comment only, but this has been reverted.